COC_PyBay.md

```diff
@@ -0,0 +1,76 @@
+ - **Be careful in the words that you choose.** We are a community of professionals, and we conduct ourselves professionally. Be kind to others. Do not insult or put down other participants, individually or as a group. Harassment and other exclusionary behavior aren't acceptable. This includes, but is not limited to:
+
+     - Violent threats or language directed against another person or yourself.
+     - Discriminatory jokes and language.
+     - Posting sexually explicit or violent material.
+     - Posting (or threatening to post) other people's personally identifying information ("doxing").
+     - Personal insults, especially those using racist or sexist terms.
+     - Unwelcome sexual attention.
+     - Deliberate intimidation, stalking, or following.
+     - Harassing photography or recording, or photographing or recording people who have explicitly declined to be recorded.
+     - Inappropriate physical contact.
+     - Advocating for, or encouraging, any of the above behavior.
+     - Repeated harassment of others. In general, if someone asks you to stop, then stop.
+
+##Where does the code of conduct apply
+
+This code of conduct applies to all spaces managed by PyBay. This includes:
+
+ - Conferences (including social events and peripheral activities, both virtual and in-person)
+ - Unconferences and sprints
+ - Meetups
+ - Workshops
+ - Presentation materials used in talks or sessions
+ - Slack
+ - Mailing lists
+ - GitHub
+ - Social media hashtag
+ - meetup.com discussion boards
+ - Any other forums created by the which the community uses for communication.
+
+In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
+Speakers, presenters, and open space organizers
+
+Speakers, presenters, and those organizing open spaces or similar events are expected to uphold the community guidelines. Please let a member of staff know immediately if there are any issues during one of these events. Presentations, open spaces, and similar events will generally not be stopped for one-time gaffes or minor problems, although a staff member will speak to the presenter afterward and we will include any incidents in our transparency report. However, staff will take immediate action to politely and calmly stop any presentation or event that repeatedly or seriously violates our code of conduct policy.
+
+###Sponsors, affiliates, and exhibitors
+
+When you sponsor a PyBay event, we welcome you as a member of our community, and we expect you to be respectful to the community you operate within.
+
+All exhibitors in the expo hall, sponsor or vendor booths, or similar activities are also subject to the code of conduct. In particular, exhibitors should not use sexualized images, activities, or other material. Booth staff (including volunteers) must not use sexualized clothing/uniforms/costumes, or otherwise create a sexualized environment.
```

Comment:
This capitalization (`code of conduct`) is not consistent with the capitalization elsewhere (`Code of Conduct`).

Reply:
change to titlecase